src/common/src/lib.rs

```diff
@@ -97,6 +97,7 @@ pub mod test_prelude {
     pub use super::array::{DataChunkTestExt, StreamChunkTestExt};
     pub use super::catalog::test_utils::ColumnDescTestExt;
 }
+pub mod secret;
```

Comment:
It might be better to use a separate crate for `secret`. 

For ease of use, you can still re-export `pub use risingwave_common_secret as secret` here.

Reply:
Done.